v1/pipeline/input.yaml

```diff
@@ -1,5 +1,6 @@
 title: Input
 type: object
+discriminator: type
```

Comment:
what does discriminator keyword do?

Reply:
Is it defined in json schema spec

Reply:
this input.yaml is like an abstract class for us and the discriminator's value (type in this case) is the field that will determine the object type